yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,151 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+# yellowbrick.model_selection.CVScores
+#
+#
+# Author:   Prema Damodaran Roman
+
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: CVScores.py [7f47800] pdamo24@gmail.com $
+
+##########################################################################
+## Imports
+##########################################################################
+
+import warnings
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+from yellowbrick.base import ModelVisualizer
+from sklearn.model_selection import cross_val_score
+
+##########################################################################
+## CVScores Visualizer
+##########################################################################
+
+class CVScores(ModelVisualizer):
+    """
+    CVScores displays cross validation scores as a bar chart and the 
+    average of the scores as a horizontal line
+    """
+    def __init__(self, model, ax=None, cv=None, scoring=None, **kwargs):    
+        """
+        Initialize the visualization with many of the options required
+        in order to make most visualizations work.
+
+        These parameters can be influenced later on in the visualization
+        process, but can and should be set as early as possible.
+
+        Parameters
```

Comment:
I think the parameters docstring should be moved out of the __init__ function and placed just above... See how @bbengfort set his up in the validation curve visualizer. 

https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/model_selection/validation_curve.py